src/Documentation/sidebar.json

```diff
@@ -123,7 +123,8 @@
       "run.md",
       "status.md",
       "unlock.md",
-      "unprotect.md"
+	    "unprotect.md",
```

Comment:
tabs -> spaces?

Reply:
@shcheklein Shall I simply use spaces?

Reply:
yes! 2 spaces. Let's also mention this in the Contributing guide - after the 80 symbols rule.

Reply:
@shcheklein PEP8 recommend to use 4 spaces instead. So shall I write the overridden rule explicitly under under Coding Style Guide?

Reply:
@algomaster99 but it's not Python, it's JS :) so, PEP8 does not apply here. Historically, the code base is using 2 spaces everywhere. Don't think migrating from this would make sense now.